charts/karmada-operator/templates/karmada-operator-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if $.Values.service }}
```

Comment:
Should we assume the service config is always set? In the deployment template, we do assume it's set, but here we don't.

Reply:
Adjust the judgment logic. This servcie and serviceMoniter are deployed in the same life cycle.